sp800-63c/sec10_usability.md

```diff
@@ -85,12 +84,11 @@ Many properties of identity have implications for how users manage identities, b
 #### 10.2.2 User Perspectives of Trust and Benefits
 
 
-Many factors can influence user adoption of federated identity systems. As with any technology, users may value some factors more than others. Users often weigh perceived benefits versus risks before making technology adoption decisions. It is critical that IdPs and RPs provide users with sufficient information to enable them to make informed decisions. The concepts of trust, and tiers of trust, fundamental principles in federated identity systems, can drive user adoption. Finally, a positive user experience may also result in increased user demand for federation, triggering increased adoption by RPs.
+Many factors can influence user adoption of federated identity systems. As with any technology, users may value some factors more than others. Users often weigh perceived benefits versus risks before making technology adoption decisions. It is critical that IdPs and RPs provide users with sufficient information to enable them to make informed decisions. The concepts of trust and tiers of trust — fundamental principles in federated identity systems — can drive user adoption. Finally, a positive user experience may also result in increased user demand for federation, triggering increased adoption by RPs.
```

Comment:
I thought I had caught all the en dashes but I hadn't. Several of them in Section 6 seem to be setting off a definition from a term, which I will replace with a colon (without the leading space, obviously).